indico/modules/events/registration/models/forms.py

```diff
@@ -364,6 +364,12 @@ def sender_address(self):
         contact_email = self.event.contact_emails[0] if self.event.contact_emails else None
         return self.notification_sender_address or contact_email
 
+    @property
```

Comment:
I don't like the idea of a property triggering a query on every access.

I'd either make this a `column_property` (can be deferred to not run it all the time) or a normal method (`get_registration_count()`)

Reply:
I just saw that the other property is already done as a column property, so just do the same for the new one: https://github.com/indico/indico/blob/211375ede936dd5f2ad9dc7b2e4d15ad1b94a050/indico/modules/events/registration/models/forms.py#L418-L423